kyo-prelude/shared/src/main/scala/kyo/Parse.scala

```diff
@@ -94,6 +98,76 @@ object Parse:
                 }
         }
 
+    /** Parses a sequence of parsers in order and collects their results
+      *
+      * @param parsers
+      *   Sequence of parsers to execute sequentially
+      * @return
+      *   Chunk containing all parsed results in order. The parse branch is dropped if any parser fails
+      */
+    def inOrder[A: Flat, S](parsers: Seq[A < (Parse & S)])(using Frame): Chunk[A] < (Parse & S) =
```

Comment:
This combinator is normally named `sequence` in other parsing libraries. `Parse.inOrder` seems more descriptive.